lib/src/main/java/graphql/nadel/validation/NadelFieldValidation.kt

```diff
@@ -133,22 +143,15 @@ internal class NadelFieldValidation(
         val overallType = overallField.type.unwrapAll()
         val underlyingType = underlyingField.type.unwrapAll()
 
-        val typeServiceSchemaElement = NadelServiceSchemaElement.from(
-            service = parent.service,
-            overall = overallType,
-            underlying = underlyingType,
-        )
-
-        // This checks whether the type is actually valid content wise
-        val outputTypeResult = typeValidation.validate(typeServiceSchemaElement)
```

Comment:
Same as above, we just need to check that the type name is the same and trust type validation to do its job.